sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RetryStrategyConfiguration.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation;
+
+import com.azure.cosmos.models.CosmosRegionSwitchHint;
+
+import java.time.Duration;
+
+public final class RetryStrategyConfiguration {
+
+    private CosmosRegionSwitchHint regionSwitchHint;
+    private Duration endToEndTimeout;
```

Comment:
hmm, endToEndOperationTimeout - what this config is used for?

Reply:
Removed `endToEndTimeout` property after offline discussion.